packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -84,6 +87,38 @@ export class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs {
     super(args);
   }
 
+  public static enhanceQueryResults(results: IQueryResult): IQueryResult {
```

Comment:
Input here should be of type `IActorQueryOperationOutput`, output type is correct.